### DISCUSSION TITLE
Problems with kaboom@next

### DISCUSSION THREAD

**Discussion by kgish** [replit/kaboom#239] [General]
I thought it'd be cool using the latest and greatest beta version of kaboom by running `yarn kaboom@next`

package.json
```
"dependencies": {
    "kaboom": "^2000.0.0-beta.22"
},
```

Additionally, since I am using `Parcel` with `Typescript` I have the following defined:

```
"alias": {
    "kaboom": "./node_modules/kaboom/dist/kaboom.mjs"
},
```

With the latest stable 0.5.1 version, everything worked just fine, but now even though I am using the identical code, I am getting the notorious message:

```
Error: kaboom_1.default.start is not a function
```

Using the 0.5.1 version I had a similar problem, but it was fixed by adding the alias mentioned above, now it seems to be back to the drawing board for me.

Any ideas what I might be doing wrong and how to fix?

**Comment by slmjkdbtl** [comment]
Hi! `start()` is removed and converged with `go()` now, can you try changing it to `go()`?

There're multiple other breaking changes in the beta version, check out [`CHANGELOG.md`](https://github.com/replit/kaboom/blob/master/CHANGELOG.md)

Also I think you don't need the `alias` anymore, please lmk if it's still needed

**Comment by kgish** [comment]
Hey thanks, it works!

Sorry, wasn't aware of the name change.

I can also verify that the alias is no longer needed.

**Comment by kgish** [comment]
Also, I noticed the fonts look much better than the previous pixelated ones.

**Comment by kgish** [comment]
Sorry, my bad ... you still need include the alias in `package.json` (forgot to delete the parcel cache first).
